src/dex/fluid-dex/fluid-dex.ts

```diff
@@ -60,12 +61,21 @@ export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
     super(dexHelper, dexKey);
     this.logger = dexHelper.getLogger(dexKey);
     this.factory = new FluidDexFactory(
```

Comment:
Am I right, that FluidDexFactory doesn't notify in any way that new pool was deployed? So, techically we won't see pool until `liqudityProxy` invokes `LogOperate` event?

Reply:
even then we won't use this pool, becase `this.pools` field is not updated. So, maybe we should update `this.pools` in factory during events handling

Reply:
@KanievskyiDanylo fixed